src/containers/SubjectPage/SubjectPage.jsx

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright (c) 2016-present, NDLA.
+ *
+ *  This source code is licensed under the GPLv3 license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { PropTypes, Component } from 'react';
+import { connect } from 'react-redux';
+import Helmet from 'react-helmet';
+import { OneColumn } from 'ndla-ui';
+import * as actions from './subjectActions';
+import { getLocale } from '../Locale/localeSelectors';
+import { SubjectShape } from '../../shapes';
+import Subject from './components/Subject';
+
+
+class SubjectPage extends Component {
+  componentWillMount() {
+    const { fetchSubject, params: { subjectId } } = this.props;
```

Comment:
Er litt usikker selv, men foreslår det i hvertfall: Er det mulig å fetche data på onEnter i routes.js og gjøre om denne komponenten til en stateless komponent slik som vist i tidligere kommentarer? Typ sånn her: http://stackoverflow.com/questions/37212328/react-router-how-to-use-route-params-in-onenter

Hva tenker du @oyvinmar ?